tools/capture-test.py

```diff
@@ -295,7 +298,9 @@ def cap_to_playback(buf):
     # treat it, and it can plausibly create false positive chirp signals
     # loud enough).
     for i in range(0, len(buf), capsz):
-        frame = [scale * x for x in struct.unpack(capfmt, buf[i:i+capsz])[0:opts.chan]]
+        frame = struct.unpack(capfmt, buf[i:i+capsz]) # Decode
```

Comment:
Re-using the name `frame` over and over does not help. The comments save the day but a little bit more imagination would not hurt... At least a plural?

```suggestion
       frames = struct.unpack(capfmt, buf[idx:idx+sample_bytes]) # Decode
```


It would help for `capsz` and `i`'s names to reflect that they are byte numbers.